nodejs/week1/warmup/warmup.js

```diff
@@ -0,0 +1,37 @@
+const roundedPI = Math.round(Math.PI * 100)/100; 
+
+class Circle {
+    constructor(radius) {
+      this.radius = radius;
+    }
+    getDiameter() {
```

Comment:
Functions which names start from `get` probably should return the value instead 🙂